core/arch/arm/plat-sam/platform_config.h

```diff
@@ -30,7 +30,19 @@
 #define PLATFORM_CONFIG_H
 
 #include <mm/generic_ram_layout.h>
+#if defined(PLATFORM_FLAVOR_sama7g54_ek)
+#include <sama7g5.h>
+#define AT91C_ID_WDT     ID_DWDT_SW
+#define AT91C_ID_TRNG    ID_TRNG
+#define AT91C_ID_SECUMOD ID_SECUMOD
+#define AT91C_ID_SFR     ID_SFR
+#define AT91C_ID_SFRBU   ID_SFRBU
+
+#define GICC_SIZE        0x1000
+#define GICD_SIZE        0x1000
```

Comment:
prefer the macro definition below, not mixed with the header files inclusion.

Reply:
Moved the definitions bellow, thanks.